JSONFormat.go

```diff
@@ -39,7 +45,7 @@ func (j jsonEntry) MarshalJSON() ([]byte, error) {
 		return nil, ErrNoMessage
 	}
 
-	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, j.Message))
+	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, escapeSpecialChars(j.Message)))
```

Comment:
I strongly disagree with you on this question, and I'm happy to elaborate, I just don't think we'll change each other's mind.

When you say

>The former is dumb simple, the latter is short, but is far more complex.

I see it completely the other way around, I think the one using the library is much dumber and simpler. (That is, if we assume that the library is reliable and performant, which I think is safe to assume in this case.)